service/src/main/java/bio/terra/workspace/service/workspace/flight/CreateGcpContextFlightV2.java

```diff
@@ -80,4 +82,4 @@ public CreateGcpContextFlightV2(FlightMap inputParameters, Object applicationCon
         new UpdateDbGcpCloudContextStep(workspaceId, appContext.getGcpCloudContextService()),
         shortRetry);
   }
-}
+}
```

Comment:
nit: Not sure it matters, but no final new line

Reply:
Done. Changed Intellij preference so this doesn't happen again.